libraries/libfc/include/fc/io/raw.hpp

```diff
@@ -564,6 +567,35 @@ namespace fc {
        }
     }
 
+    template<typename Stream, typename T>
+    inline void pack( Stream& s, const boost::dynamic_bitset<T>& value ) {
+      const auto num_blocks = value.num_blocks();
+      FC_ASSERT( num_blocks <= MAX_NUM_ARRAY_ELEMENTS );
+      fc::raw::pack( s, unsigned_int((uint32_t)num_blocks) );
+
+      // convert bitset to a vector of blocks
+      std::vector<T> blocks;
+      blocks.resize(num_blocks);
+      boost::to_block_range(value, blocks.begin());
+
+      // pack the blocks
+      for (const auto& b: blocks) {
+        fc::raw::pack( s, b );
+      }
+    }
+
+    template<typename Stream, typename T>
+    inline void unpack( Stream& s, boost::dynamic_bitset<T>& value ) {
+      unsigned_int size; fc::raw::unpack( s, size );
+      FC_ASSERT( size.value <= MAX_NUM_ARRAY_ELEMENTS );
+      std::vector<T> blocks((size_t)size.value);
+      for( uint64_t i = 0; i < size.value; ++i )
+      {
```

Comment:
For consistency with above, move `{` to the line above, or remove both curly braces.

Reply:
Thanks. I moved `{` up to be consistent with the rest of the file.